server/certstore.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022-2023 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"crypto"
+	"io"
+)
+
+type CertStoreType int
+
+const _CERTMATCHBYEMPTY_ = 0
+const _CERTSTOREEMPTY_ = 0
+
+var _, _ = _CERTMATCHBYEMPTY_, _CERTSTOREEMPTY_
+
+const (
+	WindowsCurrentUser CertStoreType = iota + 1
```

Comment:
These need to be public?

Reply:
Good catch, several don't need to be.  I will re-factor.